suggestions for beginning of tuning section (#147)
mlr-org/mlr3book: bookdown/03-optimization.Rmd

## Hyperparameter Tuning {#tuning}

Hyperparameters are second-order parameters of machine learning models that, while often not explicitly optimized during the model estimation process, can have important impacts on the outcome and predictive performance of a model. 

> Typically, hyperparameters are fixed before running a model. 

Reviewer: ```suggestion
Typically, hyperparameters are fixed before training a model. 
```

Would suite more the mlr vocabulary

> However, because the output of a model can be sensitive to the specification of hyperparameters, it is often recommended to make an informed decision about which hyperparameters may yield better model performance. 

Reviewer: ```suggestion
However, because the output of a model can be sensitive to the specification of hyperparameters, it is often recommended to make an informed decision about which hyperparameter settings may yield better model performance. 
```

> In many cases, hyperparameters may be chosen _a priori_, but it can be advantageous to try different sets of hyperparameters before fitting your model on the training data. 

Reviewer: ```suggestion
In many cases, hyperparameter settings may be chosen _a priori_, but it can be advantageous to try different settings before fitting your model on the training data. 
```
